docs/GetStarted/cmake_options_and_variables.md

```diff
@@ -0,0 +1,79 @@
+# IREE-specific CMake Options and Variables
+
+This gives a brief explanation of IREE specific CMake options and variables.
+
+* `IREE_ENABLE_RUNTIME_TRACING`:BOOL
+
+  Enables instrumented runtime tracing. Defaults to `OFF`.
```

Comment:
I'd like to iterate on the layout/styling, but that doesn't have to happen in this PR.

Some ideas:

* Use headers (``### `IREE_ENABLE_RUNTIME_TRACING`:BOOL``) so we can link to individual options
* Drop bullet points
* Adjust indentation (see https://llvm.org/docs/CMake.html)
* Try using a table and some CSS
* Organize into sections (which components to build, which backends to enable, how to configure dependencies, etc.)

Reply:
Yes, I fully agree. I don't really like the current layout myself and we need to iterate on it. This is only a first quick start to document the options.